monochart/values.yaml

```diff
@@ -712,3 +712,12 @@ networkPolicy:
   #         protocol: TCP
 
 
+
+# https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/
+hpa:
+  enabled: true
+  autoscaling:
+    minReplicas: 1
+    maxReplicas: 10
+    targetCPUUtilizationPercentage: 90
+    targetMemoryUtilizationPercentage: 90
```

Comment:
Change default to `false` so that projects which don't include `hpa` won't inherit this. Note that we did same thing for `mapping`.

```suggestion
hpa:
  enabled: false
#   autoscaling:
#     minReplicas: 1
#     maxReplicas: 10
#     targetCPUUtilizationPercentage: 90
#     targetMemoryUtilizationPercentage: 90
```